README.md

```diff
@@ -418,6 +418,56 @@ npm install prosemirror-utils
    ```
 
 
+ * **`forEachCellInColumn`**`(columnIndex: number, cellTransform: fn(cell: {pos: number, node: ProseMirrorNode}) → fn(tr: Transaction) → Transaction) → fn(tr: Transaction) → Transaction`\
+   Returns a new transaction that maps a given `cellTransform` function to each cell in a column at a given `columnIndex`.
+   It will set the selection into the last cell in the column.
+
+   ```javascript
+   const emptyCell = schema => cell => tr => {
+     const $pos = tr.doc.resolve(tr.mapping.map(cell.pos + 1));
+     return emptyCellClosestToPos($pos, schema)(tr);
+   };
+   dispatch(
+     forEachCellInColumn(0, emptyCell(state.schema))(state.tr)
+   );
+   ```
+
+
+ * **`forEachCellInRow`**`(rowIndex: number, cellTransform: fn(cell: {pos: number, node: ProseMirrorNode}) → fn(tr: Transaction) → Transaction) → fn(tr: Transaction) → Transaction`\
+   Returns a new transaction that maps a given `cellTransform` function to each cell in a row at a given `rowIndex`.
+   It will set the selection into the last cell in the row.
+
+   ```javascript
+   const emptyCell = schema => cell => tr => {
+     const $pos = tr.doc.resolve(tr.mapping.map(cell.pos + 1));
+     return emptyCellClosestToPos($pos, schema)(tr);
+   };
+   dispatch(
+     forEachCellInRow(0, emptyCell(state.schema))(state.tr)
+   );
+   ```
+
+
+ * **`setCellAttrsClosestToPos`**`($pos: ResolvedPos, attrs: Object) → fn(tr: Transaction) → Transaction`\
```

Comment:
Isn't it better to use `getCellClosestToPos` and then `setAttrs` in most cases?

Reply:
Probably more useful to split the purpose of, and then rename, this function. That would make it more composeable and versatile. For example:

```javascript
setAttrs(findNearbyCell(state.selection), { background: orange });
```

This way, if we decide we only want to apply an attribute to a nearby empty cell, for example, we can just change our code to be:

```javascript
setAttrs(findNearbyCell(state.selection), { background: orange });
```

That said, I was imagining an API like:

```
remove(findNearbyColumn(state.selection))
setAttrs(findNearbyRow(state.selection))
```